includes/blocks/class-convertkit-block-product.php

```diff
@@ -75,7 +75,7 @@ public function enqueue_scripts() {
 	 */
 	public function enqueue_styles() {
 
-		wp_enqueue_style( 'convertkit-product', CONVERTKIT_PLUGIN_URL . 'resources/frontend/css/product.css', array(), CONVERTKIT_PLUGIN_VERSION );
```

Comment:
What this intentional?

Reply:
Yes; this is renamed to `button.css` (line 79), as the same styles are now used for both the Form Trigger and Product blocks.